src/v2/api/index.md

```diff
@@ -2516,6 +2516,16 @@ Used to denote a `<template>` element as a scoped slot, which is replaced by [`s
 
 - **See also:** [Dynamic Components - keep-alive](../guide/components.html#keep-alive)
 
+>New in 2.5.0+
+
+`max` maximum number of components allowed for caching (FIFO)
```

Comment:
We should add a corresponding description into the `props` section of `keep-alive`.

```md
  - `max` - number. The maximum number ... . (new in 2.5.0+)
```